spec/ruby/core/symbol/to_proc_spec.rb

```diff
@@ -12,18 +12,18 @@
     :to_s.to_proc.call(obj).should == "Received #to_s"
   end
 
-  it "produces a proc with arity -1" do
+  it "produces a proc with arity 1" do
     pr = :to_s.to_proc
-    pr.arity.should == -1
+    pr.arity.should == 1
```

Comment:
The arity should actually be `-2` which means 1 required arguments + any number of optional arguments.